docs/src/tutorials/exporting.md

```diff
@@ -160,14 +126,14 @@ highlight=false
 1. In the [Console](https://console.platform.sh/), open your project and click **{{< icon settings >}}**.
 2. Click **Project Settings {{< icon chevron >}}**.
 3. Click **Variables** and access your variable's values and settings.
-4. Store the data on your local computer, for example in an encrypted text file for sensitive tokens.
+4. Store the data somewhere secure on your computer.
 
-Note that in the Console, you can't access the value of variables that have been created using the [`--sensitive true` flag](../development/variables/set-variables.md#variable-options).
-Use the CLI instead to retrieve these values.
+Note that in the Console, you can't access the value of variables that have been [marked as sensitive](../development/variables/set-variables.md#variable-options).
+Use the CLI to retrieve these values.
 
 {{< /codetabs >}}
 
 ## What's next
 
-- Set up your [local development environment](../development/local/_index.md)
-- Migrate to [another region](../projects/region-migration.md)
\ No newline at end of file
+- Migrate data from elsewhere [into Platform.sh](./migrating.md).
```

Comment:
If you want to add the local thing, it's fine. Just add context to the start as otherwise it doesn't seem to fit. To do X, set up your local environment